CHANGELOG.md

```diff
@@ -5,6 +5,744 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
```

Comment:
Please sync `CHANGELOG.md` file with the latest from `rokwire/illinois-app` `develop` branch and add your changes below `Unreleased` section

Reply:
done